here/here_readme.md

```diff
@@ -0,0 +1,218 @@
+Is it available on Open Data? | No
+What area does the HERE dataset represent? | All of Toronto; the co-ordinate system is EPSG: 4326
+Where is it stored? | On an internal postgres database called bigdata; in several schema (here, here_analysis, here_eval and here_gis)
+Are there any naming conventions? | If you see `_##_#` at the end of a table name (like `streets_21_1`) the first number is the year, and the second number is the quarter.
+How often is it updated? | Probe data are updated every day; reference files are usually updated quarterly
+How long are the time bins? | 5 minutes
+How far back does it go? | To 2012, but 2014 data are much more accurate
+What are the limitations? | Travel times are generally calculated for time periods lasting three or more weeks; use data from 2014 onward
+What can I get? | Custom aggregations
+Are raw data available? | No
+How do I cite HERE data? | ???
+Who can I contact about HERE data? | Email us at transportationdata@toronto.ca
+
+# Detailed dataset overview
+
+## Traffic data
+
+Historical data are acquired through the Traffic Analytics download portal. Data goes back to 2012-01-01 and is aggregated in 5-minute bins. In our database the data points are stored in partitioned tables under `here.ta` (fun fact: the "ta" stands for traffic analytics)! Have a look at the [procedure for loading new data](traffic/README.md) for that including using [`data_utils`](../data_utils/), which has been extended to partition, check constraints and index this data.
+
+### Data Schema
+
+|column|type|notes|
+|------|----|-----|
+|link_dir|text| Unique link id |
+|tx|timestamp| Timestamp of obersevation (5-min bin)|
+|epoch_min|integer| Minutes of the day|
+|length|integer| Link length (m)|
+|mean|numeric| Observed mean speed|
+|stddev|numeric| Observed speed standard deviation|
+|min_spd|integer| Observed min speed|
+|max_spd|integer| Observed max speed|
+|confidence|integer| [10-40] degree to which observation depends on historical data (higher is better)|
+|pct_x|integer| Speed at the x percentile in 5% bins|
+
+For an exploratory description of coverage (or how much probe data there is) for our roads, check out [this notebook](https://github.com/CityofToronto/bdit_team_wiki/blob/here_evaluation/here_evaluation/Descriptive_eval.ipynb).
+
+#### Getting link attributes
+
+The Traffic Analytics `link_dir` is a concatenation of the `streets` layer `link_id` and a `travel direction` character (F,T). `F` and `T` represent "From" and "To" relative to each link's reference node, which is *always* the node with the lowest latitude. In the case of two nodes with equal latitude, the node with the lowest longitude is the reference node. To join `ta` data to the `here_gis.streets_att` table use the following:
+
+```sql
+JOIN here_gis.streets_att_16_1 gis ON gis.link_id = LEFT(ta.link_dir, -1)::numeric
+```
+
+#### Functional Class 5
```

Comment:
Agreed - we had discussed moving it to the gis/README.md but it would be just as out of place there. We can either remove it altogether or supplement it with into on what the functional classes are.

Reply:
Functional classes 1 to 4 are normal streets that people drive on. Functional class 5 is a little silly in that it includes everything else (like paths n such).